cmake/Modules/FindTPLMAGMA.cmake

```diff
@@ -0,0 +1,33 @@
+IF (MAGMA_LIBRARY_DIRS AND MAGMA_LIBRARIES)
+  KOKKOSKERNELS_FIND_IMPORTED(MAGMA INTERFACE LIBRARIES ${MAGMA_LIBRARIES} LIBRARY_PATHS ${MAGMA_LIBRARY_DIRS})
+ELSEIF (MAGMA_LIBRARIES)
+  KOKKOSKERNELS_FIND_IMPORTED(MAGMA INTERFACE LIBRARIES ${MAGMA_LIBRARIES})
+ELSEIF (MAGMA_LIBRARY_DIRS)
+  KOKKOSKERNELS_FIND_IMPORTED(MAGMA INTERFACE LIBRARIES amath ${MAGMA_LIB} LIBRARY_PATHS ${MAGMA_LIBRARY_DIRS})
+ELSEIF (KokkosKernels_MAGMA_ROOT)
+  SET(MAGMA_ROOT ${KokkosKernels_MAGMA_ROOT})
+  KOKKOSKERNELS_FIND_IMPORTED(MAGMA INTERFACE
+    LIBRARIES
+      magma
+      magma_sparse
+    LIBRARY_PATHS
+      ${MAGMA_ROOT}/lib
+    HEADERS
+      magmablas.h
```

Comment:
@e10harvey I remember I used "magma_v2.h" before. Did they change it in the latest MAGMA version?

Reply:
No. `magma_v2.h` still exists. `magmablas.h` here is just used by CMake for detecting paths.